src/components/SearchBar/index.js

```diff
@@ -15,9 +15,13 @@ import style from './index.module.scss';
     > search       : Pass the search value
 
     > setSearch    : To set the search value
+
+    > sourceDesign  : To Input new style in the Input Field.
+
+    > inputSize     : Put a value such as lg, md, and sm.
 */
 
-const SearchBar = ({ placeholder, search, setSearch }) => {
+const SearchBar = ({ placeholder, search, setSearch, inputSize, sourceDesign}) => {
```

Comment:
set a default value for the inputSize since it is already being used in other pages. so that you will not need to modify the other pages